src/helpers/debounce.js

```diff
@@ -0,0 +1,10 @@
+export default function debounce(fn, ms) {
```

Comment:
It's great that you created your implementation of debounce, but it's not safe to use it. Consider using this instead
https://www.npmjs.com/package/use-debounce